NuGet.props

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- NuGet package restore sources. -->
+  <PropertyGroup>
+    <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
+      https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
+      https://api.nuget.org/v3/index.json;
+      $(RestoreSources)
+    </RestoreSources>
```

Comment:
Note this new file to hold the shared sources. Since `tests/dir.props` doesn't import `dir.props` I had to have this in some other file.

This could also go in `dependencies.props`, which both dir.props files already import. At first I didn't like that, but now I'm thinking it kind of makes sense since `RestoreSources` *is* dependency-related. What do you think?

Reply:
I think it is fine to go into dependencies.props.